mail/mailer_test.go

```diff
@@ -196,23 +204,69 @@ func badEmailHandler(messagesToProcess int) connHandler {
 	}
 }
 
-func setup(t *testing.T) (*MailerImpl, net.Listener, func()) {
+// The rstHandler authenticates the client like the normalHandler but
+// additionally processes an email flow (e.g. MAIL, RCPT and DATA
+// commands). When the `connID` is <= `rstFirst` the socket of the
+// listening connection is set to abruptively close (sends TCP RST but
```

Comment:
I was gonna correct this to "abruptly" but today I learned, "abruptive" is a word and it is incredibly apt for TCP RSTs:

https://en.wiktionary.org/wiki/abruption

> 1. (archaic) A sudden termination or interruption. [First attested in the early 17th century.][1]
> 2. A sudden breaking off; a violent separation of bodies. [First attested in the mid 17th century.][1]